flask_project/campaign_manager/test/test_models.py

```diff
@@ -2,134 +2,101 @@
 from unittest import TestCase
 from campaign_manager.models.models import *
 from sqlalchemy.orm import sessionmaker
+from campaign_manager.sqlalchemy_session import engine, session, Base
 
 
+# Creates a Test Database
 def db_setUp():
-    db_location = os.environ['TESTDATABASE_URL']
-    engine = create_engine(db_location, echo=True)
-    return engine
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
 
 
 # Destroy the mock test Database
-def db_destroy(engine):
+def db_destroy(engine_test):
     for tbl in reversed(meta.sorted_tables):
         engine.execute(tbl.delete())
 
 
-# Create a session for the Database
-def create_session(engine):
-    Session = sessionmaker(bind=engine)
-    Session.configure(bind=engine)
-    session = Session()
-    return session
+db_setUp()
 
-
-engine_test = db_setUp()
-Base.metadata.create_all(engine_test)
-session = create_session(engine_test)
+user = User()
+campaign = Campaign()
+feature_type = FeatureType()
+template_feature = FeatureTemplate()
+task_boundary = TaskBoundary()
+team = Team()
+function = Function()
 
 
 """Tests for Table: User"""
 
 
-def get_mock_user(username, email):
-    mock_user = User(
-        osm_user_id=username,
-        email=email
-        )
-    session.add(mock_user)
-    session.commit()
-    return mock_user
-
-
-def delete_mock_user(user_object):
-    session.delete(user_object)
-    session.commit()
-
-
 class TestUser(TestCase):
 
     def test_01_user_name(self):
-        mock_user = get_mock_user(
-            "user",
-            "email@gmail.com"
+        mock_user = User(
+            osm_user_id="user_new",
+            email="email@gmail.com"
             )
-        self.assertEqual(mock_user.osm_user_id, "user")
+        mock_user.create()
+        self.assertEqual(mock_user.osm_user_id, "user_new")
```

Comment:
To assert that a new user has been created, I would check that the total of users are equal to 1.
or I would select one user ordered by its creation date and check that this user equals the mock user.
but there is no column `created_at` on the user.